lib/screens_web/views/v2/audio/reconstructed_alert_fullscreen_view.ex

```diff
@@ -0,0 +1,206 @@
+defmodule ScreensWeb.V2.Audio.ReconstructedAlertFullscreenView do
+  use ScreensWeb, :view
+
+  alias Screens.Alerts.Alert
+  alias Screens.Util
+
+  def render("_widget.ssml", alert) do
+    ~E|<p><%= render_banner(alert) %><%= render_alert(alert) %></p>|
+  end
+
+  def render_banner(%{
+        unaffected_routes: _unaffected_routes
+      }),
+      do: nil
+
+  def render_banner(%{
+        region: _region,
+        routes: routes
+      }) do
+    if routes |> hd() |> Map.has_key?(:headsign) do
```

Comment:
Does `routes` always have at least one element? (Just want to make sure the `hd/1` call can't fail)

Reply:
Ooooo nice! No it doesn't! Fixing